ncm-spma/src/main/perl/spma/yumng.pm

```diff
@@ -12,17 +12,14 @@ use warnings;
 use NCM::Component;
 our $EC  = LC::Exception::Context->new->will_store_all;
 our @ISA = qw(NCM::Component);
-use EDG::WP4::CCM::Path 16.8.0 qw(unescape);
+use EDG::WP4::CCM::Element qw(unescape);
```

Comment:
please revert

Reply:
It would help to describe why ;-)

Reply:
https://github.com/quattor/CCM/issues/168